packages/caliper-fabric/lib/ConnectorConfiguration.js

```diff
@@ -86,6 +86,17 @@ class ConnectorConfiguration {
 
         return null;
     }
+    /**
+     * return array of contract definitions for specific channel
+     * @param {string} channelPassed channel wanted
+     * @returns {Array} all of contract definitions for channel
+    */
+    getContractDefinitionsForChannelName(channelPassed) {
+        const channelList = this.adapterConfiguration.channels;
+        const channelWanted = channelList.filter(channelContracts => channelContracts.channelName === channelPassed);
+        const contractDefinitions = channelWanted[0].contracts;
```

Comment:
For consistency, if the contracts definition is not there, then we should just return an empty array. Currently if it wasn't present we would be returning null or undefined. This will need a further test to check for.